modules/images/webp-uploads/load.php

```diff
@@ -675,7 +675,10 @@ function webp_uploads_wepb_fallback() {
 	<?php
 	$javascript = ob_get_clean();
 
+	$sizes_with_mime_type_support = webp_uploads_get_image_sizes_additional_mime_type_support();
+
 	wp_print_inline_script_tag(
+		sprintf( 'window.wpPerfLabWebpSizes = %s;', wp_json_encode( array_filter( $sizes_with_mime_type_support ) ) ) . "\n" .
```

Comment:
Thank you, Felix, for the review and details. We only allow some sizes for additional mime types, so we should check only those sizes, not all, because other sizes do not have WebP images.

I have updated a PR that removes this code and checks whether the current attachment has the MIME type versions available or not.